Automattic/woocommerce-payments
# Fix for fatal error when saving changes on Multi-Currency page

Fixes #2180 

#### Changes proposed in this Pull Request

- Fixes a fatal error that occurs when clicking `Save Changes` on the main Multi-Currency page (with the currency list) by hiding the save changes button on this page.

#### Testing instructions





- Checkout this branch. Go to the main `Settings -> Multi Currency` page.
- Verify the `Save Changes` button is no longer there.
- Verify the `Save Changes` button is present on the "edit" page for a single currency.
- Verify the code changes make sense.

-------------------
